src/args.rs

```diff
@@ -125,6 +125,12 @@ pub struct Args {
     /// Will enforce a fresh git init on the generated project
     #[structopt(long)]
     pub force_git_init: bool,
+
+    /// Allows running system commands without being prompted.
+    /// If this flag is not set, you will be prompted before running any
```

Comment:
I would prefer this doc to be written as what happes when the flag is set!

In other words; I'd like this more:
*Warning*: Setting this flag will enable `cargo-generate` to run system commands _without_ user confirmation. Use at your own risc.